JHUGenLexicon/src/JHUGenLexiconTranslator.cc

```diff
@@ -166,10 +194,218 @@ void JHUGenLexiconTranslator::translate(){
+          voutput.at(coupl_ampjhutrip_dV_Z).first += 1.0 + (vev_lam)* 4.0/(16.0*sw-8.0) * delta_v;
+          voutput.at(coupl_ampjhutrip_dV_A).first += 1.0;
+          voutput.at(coupl_ampjhutrip_dP_Z).first += 1.0 - (vev_lam)*4.0/(4.0-8.0*sw) * delta_v;
+          voutput.at(coupl_ampjhutrip_dM_Z).first += 1.0;
+          voutput.at(coupl_ampjhutrip_dZZWpWm).first += cw/sw *(1.0 -(vev_lam)*4.0/(4.0-8.0*sw) * delta_v);
+          voutput.at(coupl_ampjhutrip_dZAWpWm).first += sqrt(cw)/sqrt(sw) * (1.0 - (vev_lam)*4.0/(4.0-8.0*sw) * delta_v);
+          voutput.at(coupl_ampjhutrip_dP_A).first = 1.0;
+          voutput.at(coupl_ampjhutrip_dM_A).first = 1.0;
+          voutput.at(coupl_ampjhutrip_dAAWpWm).first = 1.0;
+        }
+        else {
+          voutput.at(coupl_ampjhu_ghz1).first += -(vev_lam)*6.0 * delta_v + 2.0;
+          voutput.at(coupl_ampjhu_ghw1).first += 2.0 -(vev_lam)*(sw)/(cw-sw) *delta_v ;
+          voutput.at(coupl_ampjhu_ghz1_prime2).first += (vev_lam)*2.0 *delta_v ;
+          voutput.at(coupl_ampjhu_ghw1_prime2).first += (vev_lam)*pow(MW,2)/(2.0*pow(MZ,2)*(1.0-2.0*sw)) * delta_v;
+          voutput.at(coupl_ampjhu_ghzgs1_prime2).first += - (vev_lam)*(4.0*sqrt(sw)*sqrt(cw))/(cw-sw) * delta_v;
+    	}
+      }
+      if (basis_output==bEFT_JHUGen){
+        if (include_triple_quartic_gauge){
+          voutput.at(coupl_eftjhutrip_ghz1).first += -6.0 *(vev_lam)* delta_v + 2.0;
+          voutput.at(coupl_eftjhutrip_ghz1_prime2).first += 2.0*(vev_lam) * delta_v ;
+          voutput.at(coupl_eftjhutrip_dV_Z).first += (-4.0+8*sw)/(-4.0+8*sw) + (vev_lam)*4.0/(-4.0+8.0*sw) * delta_v;
+          voutput.at(coupl_eftjhutrip_dV_A).first += 1.0;
+          voutput.at(coupl_eftjhutrip_dP_Z).first += (-4.0+8*sw)/(-4.0+8.0*sw) + (vev_lam)*4.0/(-4.0+8.0*sw) * delta_v;
+          voutput.at(coupl_eftjhutrip_dM_Z).first += (-4.0+8*sw)/(-4.0+8.0*sw) + (vev_lam)*4.0/(-4.0+8.0*sw) * delta_v;
+          voutput.at(coupl_eftjhutrip_dZZWpWm).first += (cw)/(sw*(cw-sw)) - (2.0*cw)/(cw-sw) - vev_lam * (2.0*cw)/(sw*(cw-sw)) * delta_v ;
+          voutput.at(coupl_eftjhutrip_dZAWpWm).first += (2*sqrt(cw)*sqrt(sw))/(sw-cw) - (4*sqrt(cw))/(4*sqrt(sw)*(sw-cw)) + vev_lam * sqrt(cw)/(sqrt(sw)*(sw-cw)) * delta_v;
+          voutput.at(coupl_eftjhutrip_dP_A).first = 1.0;
+          voutput.at(coupl_eftjhutrip_dM_A).first = 1.0;
+          voutput.at(coupl_eftjhutrip_dAAWpWm).first = 1.0;
+	}
+        else{
+          voutput.at(coupl_eftjhu_ghz1).first += -6.0 * (vev_lam)*delta_v + 2.0;
+          voutput.at(coupl_eftjhu_ghz1_prime2).first += 2.0*(vev_lam)*delta_v ;
+        }
+      }
+    }
+  }
+  // Format for either CC or NC output
+  
+  // Formatting for Charged Current
+  if (HW_couplings_only){
+    if (basis_output==bAmplitude_JHUGen){
+      if(include_triple_quartic_gauge){
+        std::vector< std::pair<double, double> > tempoutput(17, std::pair<double, double>(0, 0));
```

Comment:
Why not use nAmplitude_... variables instead of hardcoded values? Similar in other such lines below.

Reply:
In order to prevent having to make new basis I just manually change the output vector. Otherwise the code will get too large and hard to keep track of changes. So since I didn't make new basis i cant use nAmp...